app/src/main/java/org/wikipedia/main/MainFragment.kt

```diff
@@ -486,39 +487,36 @@ class MainFragment : Fragment(), BackPressedHandler, FeedFragment.Callback, Hist
     private fun maybeShowEditsTooltip() {
         if (currentFragment !is SuggestedEditsTasksFragment && Prefs.shouldShowSuggestedEditsTooltip() &&
                 Prefs.getExploreFeedVisitCount() == SHOW_EDITS_SNACKBAR_COUNT) {
-            binding.mainNavTabLayout.postDelayed({
-                if (!isAdded) {
-                    return@postDelayed
-                }
-                Prefs.setShouldShowSuggestedEditsTooltip(false)
+            enqueueTooltip({
```

Comment:
This can be a lambda style. `enqueueTooltip { ... } `.

Reply:
Whoops, it looks like my Android Studio got corrupted and stopped showing me automatic code insights.  I've now reinstalled it.